indigo/distribution.yaml

```diff
@@ -2495,6 +2495,21 @@ repositories:
       url: https://github.com/Hacks4ROS/h4r_x52_joyext.git
       version: develop
     status: maintained
+  haf_grasping:
+    doc:
+      type: git
+      url: https://github.com/davidfischinger/haf_grasping.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/davidfischinger/haf_grasping/tree/v0.5-beta
```

Comment:
This is not a proper release repository. Please follow the bloom tutorials (http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) to release your package or remove the release entry and only keep doc and source.